spec/integration/posts_index_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
```

Comment:
- [ ] Kindly implement some tests for the login page, you can use the following method to fill in the inputs in a page:
```
fill_in 'Email', with: 'user1@gmail.com'
```

Reply:
Hello @ShadyShawkat,
For this milestone we are not supposed to implement the tests for login page as we are on the new version of the RoR Module where they messed with the order of the milestones. That's the reason we are including the specs for the four views and not the login. if you need any clarification, please reach me on slack.

Reply:
Hey @levy002, 

Are [these](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md) the requirements for this milestone?

Reply:
@ShadyShawkat The Rails module is updated and the authentication project is after this one. The students don't have login pages yet so I think they can't create tests for the login page. The requirements of this project should be updated. 😄